src/test/kotlin/CalculatorTest.kt

```diff
@@ -8,28 +9,37 @@ class CalculatorTest {
     @Test
     fun calculate() {
         val calculator = Calculator()
-        val input = listOf<String>("4", "+", "4", "/", "2", "*", "2")
-        val calcValue = calculator.calc(input)
+        val input = listOf("4", "+", "4", "/", "2", "*", "2")
+        val calcValue = calculator.calculate(input)
 
-        assert(calcValue == 8.0)
+        assertThat(calcValue).isEqualTo(8.0)
     }
 
-    @DisplayName("계산기 비 정상 테스트")
+    @DisplayName("계산기 테스트(단일 숫자 입력)")
     @Test
-    fun calculateExcept() {
+    fun calculateOnlyOneParameter() {
         val calculator = Calculator()
+        val inputOne = listOf("4")
+        val oneValue = calculator.calculate(inputOne)
+        assertThat(oneValue).isEqualTo(4.0)
+    }
 
-        val inputOne = listOf<String>("4")
-        val oneValue = calculator.calc(inputOne)
-        assert(oneValue == 4.0)
+    @DisplayName("계산기 테스트(단일 숫자 & 단일 연산자 입력)")
+    @Test
+    fun calculateOnlyTwoParameter() {
+        val calculator = Calculator()
+        val inputTwo = listOf<String>("4", "-")
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-racingcar/pull/28#discussion_r460396741

```suggestion
        val inputTwo = listOf("4", "-")
```

Reply:
[e95d594](https://github.com/next-step/kotlin-racingcar/pull/80/commits/e95d594f015a94871e57b1b8c833e17b9fe4782b) 수정 반영 하였습니다.
테스트 코드를 수정하여 복붙하는 과정에서 문제가 발생한거 같습니다.
커밋전에 다시한번 코드 확인을 해보도록 하겠습니다!